scripts/source-maintenance.sh

```diff
@@ -118,12 +118,12 @@ for FILENAME in `git ls-files`; do
 	#
 	# If a file includes openssl headers, then it must include compat/openssl.h
 	#
-	if test "${FILENAME}" != "compat/openssl.h" && \
-	   (FI=`grep "#include.*openssl/" "${FILENAME}" 2>/dev/null`; \
-	    test "x${FI}" != "x") && \
-	   (FI=`grep "#include \"compat/openssl\.h\"" "${FILENAME}" 2>/dev/null`; \
-	    test "x${FI}" == "x"); then
-		echo "ERROR: ${FILENAME} includes openssl headers without including \"compat/openssl.h\""
+	if test "${FILENAME}" != "compat/openssl.h"; then
+		FA=`grep "#include.*openssl/" "${FILENAME}" 2>/dev/null | head -1`;
+		FB=`grep "#include.*compat/openssl" "${FILENAME}" 2>/dev/null | head -1`;
+		if test "x${FA}" != "x" && test "x${FB}" = "x"; then
```

Comment:
```suggestion
		if test "x${FA}" != "x" -a "x${FB}" = "x"; then
```

... for consistency with other similar "a and b" tests in this script.